Gems/EMotionFX/Code/EMotionFX/Rendering/OpenGL2/Source/Material.h

```diff
@@ -28,18 +28,18 @@ namespace RenderGL
             mNumTriangles = 0;
             mNumVertices  = 0;
 
-            mNodeIndex     = MCORE_INVALIDINDEX32;
+            mNodeIndex     = InvalidIndex;
             mMaterialIndex = MCORE_INVALIDINDEX32;
```

Comment:
question: is `MCORE_INVALIDINDEX32` meant to still be used?

Reply:
Yes. Unfortunately that is still a little messy. The goal here was to only change variables/members to `size_t` if they are used as an index to some other thing which is now stored in an `AZStd::vector` instead of an `MCore::Array`. `mNodeIndex` here is being changed to a `size_t` because it represents an index to be passed to `EMotionFX::Skeleton::GetNode(size_t)`. `mMaterialIndex` does not, so it stays as a `uint32`, and its invalid index value has not changed.